lib/created_message_queue_handler.ts

```diff
@@ -246,39 +269,46 @@ export function processResolve(
   messageContent: MessageContent,
   senderMetadata: CreatedMessageEventSenderMetadata
 ): void {
-  if (isRight(errorOrNotification)) {
-    const notification = errorOrNotification.value;
-    // TODO: handle all notification channels
-    // the notification object has been created with an email channel
-    // we output a notification event to the email channel queue
-    if (notification.emailNotification) {
-      // tslint:disable-next-line:no-object-mutation
-      context.bindings.emailNotification = {
-        messageContent,
-        messageId: notification.messageId,
-        notificationId: notification.id,
-        senderMetadata
-      };
+  if (isLeft(errorOrNotification)) {
```

Comment:
minor style improvement, just after you verify that the `Either` is a left, you can extract its left value, i.e.:

```ts
if (isLeft(errorOrNotification)) { 
  const error = errorOrNotification.value;
  ...
```

and you'll save the duplicated assignment (274 and 286)

Reply:
done